demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -572,24 +572,28 @@ def _collect_layouts_dependencies(pack_layouts: list,
         for layout in pack_layouts:
             layout_data = next(iter(layout.values()))
             layout_dependencies = set()
-            if layout_data.get('group') == 'indicator' or layout_data.get('kind') == 'indicatorsDetails':
+            if layout_data.get('definitionId') not in [None, 'incident', 'indicator']:
```

Comment:
If the schema will enforce the `definitionId` key then the `None` value is not possible (almost certain of it), but anyhow it can be an empty string or any "empty" data.